Clustering/KMeans/src/main/java/org/tribuo/clustering/kmeans/TrainTest.java

```diff
@@ -54,6 +55,9 @@ public String getOptionsDescription() {
         public int iterations = 10;
         @Option(charName='d',longName="distance",usage="Distance function to use in the e step. Defaults to EUCLIDEAN.")
         public Distance distance = Distance.EUCLIDEAN;
+        @Option(charName='i',longName="initialisation",usage="Type of initialisation to use for centroids. Defaults " +
```

Comment:
There already is a `i` argument, so this options class is invalid, could you change it to something else?

Reply:
Also, there's no need to put the default into the usage string, I updated the options code to print that automatically. While you're in here could you remove the default bit from the other usage strings? I missed that in the docs pass I did.